src/mlpack/methods/amf/termination_policies/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 set(SOURCES
   simple_residue_termination.hpp
   simple_tolerance_termination.hpp
-  validation_rmse_termination.hpp
+  validation_RMSE_termination.hpp
```

Comment:
Hmm, none of the other source files in mlpack have capital letters---do you think we could change the file itself to `validation_rmse_termination.hpp` instead?